File: server/FlightLog-server/src/main/java/bitc/fullstack503/flightlog/flightlogserver/dto/flightUserDTO.java
package bitc.fullstack503.flightlog.flightlogserver.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class flightUserDTO {
  @JsonProperty("flightUserId")
  private String flightUserId;

  @JsonProperty("flightUserFirstname")
  private String flightUserFirstname;

  @JsonProperty("flightUserLastname")
  private String flightUserLastname;

  @JsonProperty("flightUserKoFirstname")
  private String flightUserKoFirstname;

  @JsonProperty("flightUserKoLastname")
  private String flightUserKoLastname;

  @JsonProperty("flightUserPw")
  private String flightUserPw;

  @JsonProperty("flightUserBirth")
  private String flightUserBirth;

  @JsonProperty("flightUserPhone")
  private String flightUserPhone;

  @JsonProperty("flightUserGender")
  private String flightUserGender;

  @JsonProperty("flightUserEmail")
  private String flightUserEmail;
}

File: app/app/src/main/java/bitc/fullstack/FlightLog/dto/flightUserDTO.kt
package bitc.fullstack.FlightLog.dto

import com.google.gson.annotations.SerializedName

data class flightUserDTO(
  @SerializedName("flightUserId")
  var flightUserId: String,

  @SerializedName("flightUserFirstname")
  var flightUserFirstname: String,

  @SerializedName("flightUserLastname")
  var flightUserLastname: String,

  @SerializedName("flightUserKoFirstname")
  var flightUserKoFirstname: String,

  @SerializedName("flightUserKoLastname")
  var flightUserKoLastname: String,

  @SerializedName("flightUserPw")
  var flightUserPw: String,

  @SerializedName("flightUserBirth")
  var flightUserBirth: String,

  @SerializedName("flightUserPhone")
  var flightUserPhone: String,

  @SerializedName("flightUserGender")
  var flightUserGender: String,

  @SerializedName("flightUserEmail")
  var flightUserEmail: String
)